src/core/src/op/add.cpp

```diff
@@ -78,6 +78,17 @@ bool op::v1::Add::evaluate(const HostTensorVector& outputs, const HostTensorVect
     return add::evaluate_add(inputs[0], inputs[1], outputs[0], get_autob());
 }
 
+bool op::v1::Add::evaluate(ov::TensorVector& outputs, const ov::TensorVector& inputs) const {
+    OV_OP_SCOPE(v1_Add_evaluate);
+    if (std::none_of(inputs.cbegin(), inputs.cend(), [](const ov::Tensor& t) {
+            return is_vector(t.get_shape()) && t.get_shape().front() == 0;
+        })) {
+        return BinaryElementwiseArithmetic::evaluate(outputs, inputs);
```

Comment:
`BinaryElementwiseArithmetic` doesn't override `evaluate`, so it will call `Node::evaluate` that converts Tensors to HostTensors and finally will run old `v1::Add::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs)`. 
Is that a temporary workaround?

Reply:
Is temporary when will be possible to migrate from host tensor to ov tensor and not keep both version will be not required. Just to avoid duplication for now.